apps/wso2-digital-wallet/src/pages/RecoverWallet/RecoverWallet.css

```
@@ -7,25 +7,295 @@
-  min-height: auto;
-  max-height: none;
-  overflow-y: visible;
-  padding-bottom: 40px;
+.rw-seed-input {
+  flex: 1;
+  min-width: 0;
+  width: 0;
+  border: none;
+  background: transparent;
+  outline: none;
+  font-family: "JetBrains Mono", "Monaco", "Menlo", "Ubuntu Mono", monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)